packages/typescript/chat-core/test/ai/use-ai-chat.test.ts

```
@@ -0,0 +1,406 @@
+          data: { test: true },
+        },
+      ];
+
+      expect(context[0].priority).toBeUndefined();
+      // Default priority should be handled in the hook implementation
+    });
+  });
+
+  describe('error handling', () => {
+    it('should create proper error types', () => {
+      const { AIError, AI_ERROR_CODES } = require('../../src/ai/types');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)